components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java

```diff
@@ -72,7 +82,11 @@ public static void createOAuth2Application(String applicationName, String portal
         oAuthConsumerAppDTO.setOAuthVersion(VERSION_2);
         oAuthConsumerAppDTO.setOauthConsumerKey(consumerKey);
         oAuthConsumerAppDTO.setOauthConsumerSecret(consumerSecret);
-        oAuthConsumerAppDTO.setCallbackUrl(IdentityUtil.getServerURL(portalPath, false, true));
+        String callbackUrl = IdentityUtil.getServerURL(portalPath, false, true);
```

Comment:
Are we using this util your imple, @omindu ?

Reply:
yes, this will return the tenant qualified URL with the `portalPath`